cypress/integration/userCanReadAnArticle.feature.js

```diff
@@ -0,0 +1,44 @@
+describe("User can read an article", () => {
+  beforeEach(() => {
+    cy.intercept("GET", "**api/articles/", {
+      fixture: "indexRespondsFromApi.json",
+      statusCode: 200,
+    }).as("indexApiGetRequest");
+
+    cy.intercept("GET", "**/api/articles/0", {
+      fixture: "articleExample.json",
+    }).as("showApiGetRequest"); 
+
+    cy.visit("/");
+    cy.get("[data-cy=overall-page]").children().first().children().first().children().first().click();
```

Comment:
Totally unreadable step. What is it you are clicking?

Reply:
Got it work with the direct call out, originally wouldnt work that way and then never updated